docker/utils.py

```diff
@@ -33,12 +34,12 @@ def override_connect_configs(confs):
   for k,v in confs.items():
     #remove CONNECT_ prefix
     k = k[8:]
-    key = k.lower().replace("_",".")
+    key = k.lower().replace("_",".") + "="
       
     # override connect_conf with env_vars
     for i in range(0, len(connect_conf)):
       if connect_conf[i].startswith(key):
-        connect_conf[i] = key + "=" + v
+        connect_conf[i] = key + v
```

Comment:
Fix for when properties name started in the same way (ie: key.converter and key.converter.schema.registry.url).